lib/item.rb

```diff
@@ -1,10 +1,8 @@
 class Item
-  attr_reader :name, :description, :price, :code, :count
```

Comment:
This was a good change! I think that the item class doesn't need to know about count and code